packages/core/src/Provider/TxSubmitProvider/types.ts

```diff
@@ -2,10 +2,14 @@
 // @ts-ignore
 import Cardano, { Provider } from '../..';
 
+export interface SubmitTxArgs {
+  signedTransaction: Uint8Array;
```

Comment:
Not originally in scope of this ticket, but since we're making breaking changes perhaps we should change this to `type SerializedTransaction = Cardano.util.HexBlob`. Easier to use in http api. Thoughts @rhyslbw ?

Reply:
Agree. Fine by me as a separate commit